### DISCUSSION TITLE
how do I use pile

### DISCUSSION THREAD

**Discussion by lostmonster** [UdaraJay/Pile#103] [General]
opened it up, nothing works in the interface. nothing is clickable etc. 
<img width="1028" alt="Screenshot 2024-10-10 at 1 40 50 PM" src="https://github.com/user-attachments/assets/b64d3e28-72d0-4d2e-9cab-44d6a20a944f">

**Comment by koenbroumels** [comment]
downloaded it yesterday, installed and worked for me.. only the ai integration doesn't seem to work for me. 
i'm working on windows by the way.
